.vscode/tags

```diff
@@ -0,0 +1,56 @@
+!_TAG_FILE_FORMAT	2	/extended format; --format=1 will not append ;" to lines/
```

Comment:
I would add this to your .gitignore if it's getting pushed in with your commits...alternatively, I wonder if you do ```git add .``` when you add your changes. That's okay, but it adds the "."-prepended files to your commits too. You should use ```git add *``` to add all changed files except for dot-prepended ones.